### PR TITLE
BROKERS: Get Labs Data

### PR DESCRIPTION
**Pull request by AndrewMcClelland**
Closes Task #39590625: BROKERS: Get Labs Data

### CODE REVIEW

File: DMX.Core.Api/Models/Devices/DeviceCommand.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace DMX.Core.Api.Models.Devices
+{
+    public class DeviceCommand
```
Comment (hassanhabib) [inline DMX.Core.Api/Models/Devices/DeviceCommand.cs:9]: Let's distinguish these external models from our localized models.
This is not something I mentioned in the video. but here's a visualization.

```mermaid
flowchart RL
    A[External Device] --> B{{Device Foundation Service}} --> Device
```

The one thing I'm hoping you could educate me on is, how do we know which devices belong to the same NUK/Host? Is there some common ID somewhere?

Comment (AndrewMcClelland) [reply DMX.Core.Api/Models/Devices/DeviceCommand.cs]: > Let's distinguish these external models from our localized models. This is not something I mentioned in the video. but here's a visualization.

Updated external models naming to prefix with `External`.

> The one thing I'm hoping you could educate me on is, how do we know which devices belong to the same NUK/Host? Is there some common ID somewhere?

Discussed in standup.

**Comment by AndrewMcClelland** [comment]
Wrote a little stub test in the HomeController (that is not committed) to test hitting the live endpoint to get available devices and it worked :)

File: DMX.Core.Api/Brokers/ReverbApis/ReverbApiBroker.Devices.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        {
+            const string RelativeUrl = "api/GetAvailableDevicesAsync";
+
+            ExternalLabsServiceInformation ReverbServiceProperties = new ExternalLabsServiceInformation
+            {
+                ServiceType = ReverbServiceType,
+                ServiceId = reverbServiceId
+            };
+
+            return await this.PostAync<ExternalLabsServiceInformation, ExternalLabsCollection>($"{RelativeUrl}?code={this.accessKey}", ReverbServiceProperties);
```
Comment (hassanhabib) [inline DMX.Core.Api/Brokers/ReverbApis/ReverbApiBroker.Devices.cs:24]: We need to break this into < 120 chars line.
Also use named parameters when the input doesn't match in part or in full the parameter name.
https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Methods.md#104-passing-parameters

**Pull request merged**